docs/docs_server_configuration_reference.md

```diff
@@ -308,7 +308,7 @@ how they should be spawned: in interior or exterior. By default all the npcs are
 
 ## weaponStaminaModifiers
 
-This option allows you to flexibly adjust stamina forfeits of players' attacks using keywords set in Creation Kit.
+This option allows you to flexibly adjust stamina forfeits of players' attacks using keywords set in the Creation Kit.
```

Comment:
Please also let users know that it's (at this time) only active for servers with SweetPie.esp loaded.